src/pretix/plugins/paypal/payment.py

```diff
@@ -171,16 +171,10 @@ def settings_content_render(self, request):
         if self.settings.connect_client_id and not self.settings.secret:
             # Use PayPal connect
             if not self.settings.connect_user_id:
-                settings_content = (
-                    "<p>{}</p>"
-                    "<a href='{}' class='btn btn-primary btn-lg'>{}</a>"
-                ).format(
-                    _('To accept payments via PayPal, you will need an account at PayPal. By clicking on the '
-                      'following button, you can either create a new PayPal account connect pretix to an existing '
-                      'one.'),
-                    self.get_connect_url(request),
-                    _('Connect with {icon} PayPal').format(icon='<i class="fa fa-paypal"></i>')
-                )
+                # Migrate User to PayPal v2
+                self.event.disable_plugin("pretix.plugins.paypal")
+                self.event.enable_plugin("pretix.plugins.paypal2")
+                self.event.save()
```

Comment:
Interesting hack to put this in the settings_content_render, but how will this look for users? Will they see a blank PPv1 form and then manually need to go back and find that they now have the PPv2 form? Might be very confusing UX?